src/_components/Dashboard/DashboardContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { getAssetStatus } from '../../_actions/assetStatus.action';
+
+import DashboardComponent from '../../components/Dashboard/DashboardComponent';
+
+class DashboardContainer extends PureComponent {
```

Comment:
This is a redundant component as it is just passing props to `DashboardComponent`. Please delete it